src/Affects/Connections/Events/Drivers/Configuring.php

```diff
@@ -40,8 +40,6 @@ class Configuring
 
     public function __construct(Tenant $tenant, array &$configuration, ProvidesConfiguration $provider)
     {
-        $configuration = $this->defaults($tenant, $configuration);
-
         $this->tenant = $tenant;
         $this->configuration = &$configuration;
```

Comment:
I'm admittedly not WELL-versed in PHP's by-reference behavior... It's passed into the method by-reference, but I don't quite understand what this line is doing.  Is the `&` needed?

Reply:
It passes the variable by reference to make sure that all adjustments are then moved to the actually passed variable. Eventually, these end up in the DB Drivers.